src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -254,25 +253,25 @@ Amplify.configure(awsconfig);
 ```
 
 This will bootstrap your app to use Amplify with the deployed backend resources.
-
+ 
 **Recap:** You now have set up and configured your backend resources for Amplify authentication and are ready to start working on your front-end application.
 
 ## Determine your Auth integration path
 
 Before you implement Auth on your front-end application, you will want to evaluate the options of using the Authenticator UI component, Amplify Libraries methods, or Cognito Hosted UI for your use case. We recommend using Authenticator as the easiest to implement. You can also use Authenticator together with Amplify Libraries methods to extend capabilities when needed. Additionally, the Hosted UI is also best used with the Amplify Libraries because they provide a method to redirect to the Hosted UI.
-
+ 
 ### Compare implementation options
 
 There are a few options to implement Auth depending on how much customization you will need:
 
 |  | Amplify Authenticator | Amplify Libraries | Amplify Hosted UI |
 |:--|:--|:--|:--|
 | Description | Open source drop-in UI component for authentication | Low-level building blocks for implementing authentication | Pre-built sign-up and sign-in UI components hosted by Cognito outside of your application | 
-| Benefits | Automatically integrates with your existing Amplify configuration and allows you to easily add the entire authentication flow to your application. You simply add the component to your app and pass it your Amazon Cognito authentication configuration. You can then customize themes to adjust colors and styling as needed. | Gives you full control over the UI and logic implementation. | Cognito Hosted page with basic customization via properties. |
+| Benefits | Automatically integrates with your existing Amplify configuration and allows you to easily add the entire authentication flow to your application. You simply add the component to your app and pass it your Amazon Cognito authentication configuration. You can then customize themes to adjust colors and styling as needed. | Gives you full control over the UI and logic implementation. | Cognito Hosted page with basic customization via properties. | 
```

Comment:
I removed "Allows you to quickly add production-ready authentication flows to web and mobile applications." for Authenticator as this section was getting wordy but noting in case that sentence needs to be added back.

Reply:
sounds good.